config/config.go

```diff
@@ -1070,11 +1084,23 @@ func (cfg *ConsensusConfig) Precommit(round int32) time.Duration {
 }
 
 // Commit returns the amount of time to wait for straggler votes after receiving +2/3 precommits
-// for a single block (ie. a commit).
+// for a single block (i.e., a commit).
 func (cfg *ConsensusConfig) Commit(t time.Time) time.Time {
 	return t.Add(cfg.TimeoutCommit)
 }
 
+// CommitWithCustomTimeout is identical to Commit. However, it calculates the time for commit using the supplied customTimeout.
+// If customTimeout is 0, the default TimeoutCommit is used.
```

Comment:
I will apply your comment to further clarify. However, in my pov, since this was a method of `ConsensusConfig`, default refers to what is in the consensus config object.

Reply:
Clarified in https://github.com/celestiaorg/celestia-core/pull/1494/commits/85e92f30bb1f911e88afc50f0f84a6699d2b2bc8